### DISCUSSION TITLE
Form changed/touched/dirty

### DISCUSSION THREAD

**Discussion by bkniffler** [mantinedev/mantine#523] [Feedback]
Any chances for form changed/touched/dirty state? Would be awesome to check if a save button should be disabled because no changes have been made so far.

I could try and submit a PR :)

**Comment by rtivital** [comment]
Can you describe what API would you expect from this feature?

**Reply by bkniffler** [reply]
Kinda like this one: https://react-hook-form.com/api/useformstate

**Reply by rtivital** [reply]
Sounds fine, feel free to submit a PR

**Comment by ioson** [comment]
Any progress here?
I would love to replace Formik with Mantine's use-form and all that is missing is the dirty/touched property.

For reference: [https://formik.org/docs/api/formik#dirty-boolean](https://formik.org/docs/api/formik#dirty-boolean)

**Comment by tianhuil** [comment]
Would also love to see this!

**Comment by zhulien-ivanov** [comment]
Is someone working on this currently?

**Comment by akhil-naidu** [comment]
This feature could make this project a one-stop for frontend UI/UX.

**Comment by zeestack** [comment]
Is someone working on this?

**Comment by Cliftonz** [comment]
Looks like mantine does support this.
https://mantine.dev/form/status/
